### PR TITLE
Remove timelock expiration

### LINKED ISSUE #255: remove expiration mechanism
https://morpholabs.slack.com/archives/C02NE4SAE4B/p1698052613290089

### PR DESCRIPTION
**Pull request by MerlinEgalite**
Fixes #255

### CODE REVIEW

File: src/MetaMorpho.sol [hunk 1]
```diff
@@ -153,12 +153,9 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     /// - there's no pending value;
     /// - the timelock has not elapsed since the pending value has been submitted;
     /// - the timelock has expired since the pending value has been submitted.
-    modifier withinTimelockWindow(uint256 submittedAt) {
+    modifier afterTimelock(uint256 submittedAt) {
```
Comment (QGarchery) [inline src/MetaMorpho.sol:156]: We should also remove the comment just above this line about the timelock expiration

**Review by QGarchery — CHANGES_REQUESTED** [review]

File: src/MetaMorpho.sol (hunk 1 above)
Comment (MerlinEgalite) [reply src/MetaMorpho.sol:156]: Good catch!

**Review by Rubilmax — APPROVED** [review]

**Review by MerlinEgalite — COMMENTED** [review]
Update `README`

**Review by Rubilmax — APPROVED** [review]

**Review by Jean-Grimal — APPROVED** [review]

**Review by QGarchery — APPROVED** [review]
I still think that the behavior of changing ETA of pending values when changing the timelock is not expected (see #267). But let's consider that it is a separate issue.

LGTM otherwise

**Comment by MerlinEgalite** [comment]
> I still think that the behavior of changing ETA of pending values when changing the timelock is unexpected (see #267). But let's consider that it is a separate issue.
> 
> LGTM otherwise

To me it should be in another PR on top

**Review by Rubilmax — APPROVED** [review]

**Review by Jean-Grimal — APPROVED** [review]

**Pull request merged**